### PR TITLE
Improved metadata

### PR DESCRIPTION
**Pull request by holybaechu**

### CODE REVIEW

**Comment by holybaechu** [comment]
> and the name/description and especially the url need to be dynamic depending on the page

Going to do that tomorrow with SEO

**Comment by RappyTV** [comment]
Alright. I'm adding a few commits soon which improve the feature descriptions a bit

**Comment by RappyTV** [comment]
you can draft prs now btw :)

**Comment by RappyTV** [comment]
I get this error when trying to build/preview
```
failed to load config from D:\...\Boost-Frontend\vite.config.ts
error when starting preview server:
Error: Dynamic require of "file:///D:/.../Boost-Frontend/node_modules/dotenv/lib/main.js" is not supported
    at file:///D:/.../Boost-Frontend/vite.config.ts.timestamp-1708622757747-2c49a106a5579.mjs:6:9
    at file:///D:/.../Boost-Frontend/vite.config.ts.timestamp-1708622757747-2c49a106a5579.mjs:12:1
```

**Comment by holybaechu** [comment]
I was going to open vsc but you committed it first xd
also i really hate my school laptop
image

Anyways, Do we need to add some tests?

**Comment by RappyTV** [comment]
wdym with tests? btw it still does not read the .env file. or where do I have to place it?

**Comment by holybaechu** [comment]
> wdym with tests?
Something like unit tests.
> btw it still does not read the .env file. or where do I have to place it?
Looks like vite is having a stroke. `bun run build/index.js` should work fine. (run after building the project)

**Comment by RappyTV** [comment]
> Something like unit tests.

yea idk how to do that but you can set it up if you want. bun worked btw

**Comment by RappyTV** [comment]
Is it somehow possible to surpress these unnecessary errors when accessing the 404 page: 
image

**Comment by holybaechu** [comment]
It is possible through error handlers tho.

https://stackoverflow.com/a/75646108

**Comment by RappyTV** [comment]
alr imma test it when i get home in an hour

**Comment by RappyTV** [comment]
nope, still not working
also the docs look like this:
image

**Comment by holybaechu** [comment]
I just tested it on my iPhone and interestingly, It only happens when there's transition in the element.
So I just removed the transition.

**Comment by holybaechu** [comment]
Search Engine Optimization is done now.
I didn't enable AMP and disable CSR because it broke some pages. (especially pages with scripts)

Also, This PR is safe to merge now.

File: src/routes/sitemap.xml/+server.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
+const selfURL = "https://boost.rappy.tv";
```
Comment (RappyTV) [inline src/routes/sitemap.xml/+server.ts:1]: wrong domain
```suggestion
const selfURL = "https://boost.rappytv.com";
```

**Review by RappyTV — CHANGES_REQUESTED** [review]
Some things to fix:
1. There is this unnecessary scrollbar at the bottom on mobile and the white line on the right:
image
2. This white line and cutoff at the bottom:
image
3. The button text and hamburger menu text isn't centered:
image
The rest turned out beautiful :)

**Review by RappyTV — APPROVED** [review]
well I actually meant to just center the text in the buttons but that works aswell. LGTM

**Pull request merged**